packages/playground-server/package.json

```diff
@@ -21,27 +21,33 @@
     "url": "git+https://github.com/counterfactual/monorepo.git"
   },
   "scripts": {
-    "start": "PLAYGROUND_SERVER_ENV=production netlify-lambda -c webpack.config.js serve src",
-    "build": "PLAYGROUND_SERVER_ENV=production netlify-lambda -c webpack.config.js build src",
-    "dev": "PLAYGROUND_SERVER_ENV=development netlify-lambda -c webpack.config.js serve src",
-    "test": "PLAYGROUND_SERVER_ENV=development jest"
+    "start": "netlify-lambda -c webpack.config.js serve src",
+    "build": "netlify-lambda -c webpack.config.js build src",
+    "dev": "cross-env FIREBASE_API_KEY= FIREBASE_AUTH_DOMAIN= FIREBASE_DATABASE_URL= FIREBASE_PROJECT_ID= FIREBASE_STORAGE_BUCKET= FIREBASE_MESSAGING_SENDER_ID= PLAYGROUND_SERVER_SECRET=foo PLAYGROUND_SERVER_ENV=development netlify-lambda -c webpack.config.js serve src",
+    "test": "cross-env FIREBASE_API_KEY=a FIREBASE_AUTH_DOMAIN=b FIREBASE_DATABASE_URL=c FIREBASE_PROJECT_ID=d FIREBASE_STORAGE_BUCKET=e FIREBASE_MESSAGING_SENDER_ID=f PLAYGROUND_SERVER_SECRET=foo PLAYGROUND_SERVER_ENV=development jest"
```

Comment:
I feel like these scripts are getting super long. Maybe we can find a way to shorten this? I used `dotenv-safe` in other packages to enforce the existence of the variables in the environment.

Reply:
Maybe we can do something like this: https://github.com/kentcdodds/cross-env#usage
```
{
  "scripts": {
    "parentScript": "cross-env GREET=\"Joe\" npm run childScript",
    "childScript": "cross-env-shell \"echo Hello $GREET\""
  }
}
```

Reply:
:+1: Sounds interesting. If you don't mind, I'd like to work this out on a different PR and come up with a solution for any packages that required env-vars to work.